dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -654,11 +644,19 @@ export const Card = ({
 												imagePositionOnMobile={
 													imagePositionOnMobile
 												}
+												//** TODO: IMPROVE THIS MAPPING */
 												// image size defaults to small if not provided. However, if the headline size is large or greater, we want to assume the image is also large so that the play icon is correctly sized.
```

Comment:
This comment might benefit from rewording to make it slightly clearer as what was _large_ is now _small_ in the new scale? (Although the rendered text _is_ still large 🙃)